curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/increment-a-number-with-javascript.english.md

```diff
@@ -26,13 +26,13 @@ Change the code to use the <code>++</code> operator on <code>myVar</code>.
 
 ```yml
 tests:
-  - text: <code>myVar</code> should equal <code>88</code>
+  - text: <code>myVar</code> should equal <code>88</code>.
     testString: assert(myVar === 88);
   - text: <code>myVar = myVar + 1;</code> should be changed
```

Comment:
```suggestion
  - text: <code>myVar = myVar + 1;</code> should be changed.
```

Reply:
@moT01 Can not commit suggested change "as-is" because you have deleted a space off the beginning of the line which must be there.

Reply:
I noticed that, I didn't actually delete them though - if you look at the first part of this message, the line has two spaces in front and my suggestion has two spaces in front. Not sure why the second diff is showing a third space that didn't exist.

Reply:
Hmm, if I just add a third space to my suggestion, it doesn't show that anymore. But would that add a third space to the file?

Reply:
I will trying committing this first one.  If it fails, then I will add the periods via extra commit outside GitHub.

FYI - I debated about adding periods to these because the majority of the challenges in this section do not have periods at the end of the test messages.